test/e2e/main.go

```diff
@@ -20,6 +20,14 @@ type Test struct {
 func main() {
 	logger := log.New(os.Stdout, "test-e2e", log.LstdFlags)
 
+	// latestVersion, err := testnet.GetLatestVersion()
+	// testnet.NoError("failed to get latest version", err)
```

Comment:
if this code is not used, can we delete it?

Reply:
we will use it once the BRR issue is resolved

Reply:
The `--force-no-bbr` flag exists on `main` but may not exist on v2.1.2

Reply:
thanks. the hard coded version removed after having the tests passed with the main. Since it was on a fork branch, the `07411e2` version from main is used.